apps/asap-server/src/utils/sync-google-event.ts

```diff
@@ -61,7 +61,7 @@ export const syncEventFactory = (
       endDateTimeZone: googleEvent.end.timeZone || defaultTimezone,
       status: (googleEvent.status.charAt(0).toUpperCase() +
         googleEvent.status.slice(1)) as EventStatus, // TODO: use lowercase
-      calendar: [calendarId],
+
```

Comment:
im making sure that the calendar ID does not get overwritten - although it's not necessary as there is a conditional added (Further down) to skip the update if it comes from a different calendar so only the original owner of the event would be attached